app/views/menus/_site_menu.html.erb

```diff
@@ -0,0 +1,6 @@
+<li><%= link_to 'Dashboard', root_path %></li>
+<li><a href="#">Co-ops</a></li>
+<li><a href="#">Loans</a></li>
+<li><a href="#">Calendar</a></li>
+<li><a href="#">Members</a></li>
+<li><a href="#">Reports</a></li>
```

Comment:
I like how you have separate files for logically separate pieces that are likely to change. That'll make it much easier. I think the typical pattern is to put all layout files inside the layouts folder. `views/menus` looks to me like it should hold views related to a menu model. Can you move the menus and page folders into the layouts folder?